src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -1013,6 +1013,39 @@ namespace cryptonote
   };
 
     
+  //-----------------------------------------------
+  struct COMMAND_RPC_GET_NET_STATS
+  {
+    struct request_t
+    {
+
+      BEGIN_KV_SERIALIZE_MAP()
+      END_KV_SERIALIZE_MAP()
+    };
+    typedef epee::misc_utils::struct_init<request_t> request;
+
+
+    struct response_t
+    {
+      std::string status;
```

Comment:
It's convention. It's sometimes used to convey "OK, but you may want to know about this" information. The wallet does not have one.

Reply:
Yes, I just did for my new call what everybody did with the existing calls already, so while it may or may not make sense it's at least kept consistent.